packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -290,11 +288,9 @@ async function createNativeProjectsFromTemplateAsync(projectRoot: string): Promi
   exp.ios = exp.ios ?? {};
   exp.ios.bundleIdentifier = bundleIdentifier;
 
-  // TODO: remove entryPoint and log about it for sdk 37 changes
-  if (exp.entryPoint && exp.entryPoint !== EXPO_APP_ENTRY) {
-    log(`- expo.entryPoint is already configured, we recommend using "${EXPO_APP_ENTRY}`);
-  } else {
-    exp.entryPoint = EXPO_APP_ENTRY;
+  if (exp.entryPoint) {
+    delete exp.entryPoint;
+    log(`- expo.entryPoint is not needed and has been removed.`);
   }
```

Comment:
I've modified this so the entryPoint is simply deleted upon ejection. In a static config, this should be persisted, otherwise it won't matter because every eject will delete the field.